docs/en/advanced/Alluxio-Storage-Management.md

```diff
@@ -412,6 +414,74 @@ $ bin/alluxio runTests -Dalluxio.user.file.create.ttl=3m -Dalluxio.user.file.cre
 For this example, ensure the `alluxio.master.ttl.checker.interval` is set to a short
 duration, such as a minute, in order for the the master to quickly identify the expired files.
 
+## Managing Data Replication in Alluxio
+
+### Passive Replication
+
+Like many distributed file systems, each file in Alluxio consists of one or multiple blocks stored
+across the cluster. By default, Alluxio may adjust the replication level of different blocks
+dynamically and automatically based on the workload and storage capacity. For example, Alluxio may
+create more replicas of a particular block when more clients request to read this block with read
+type `CACHE` or `CACHE_PROMOTE`; Alluxio may also remove existing replicas when they are used less
+often to reclaim the space for data that is accessed more often ([Evictor in Alluxio Storage](#eviction-policies)).
+It is possible that in the same file different blocks have a different number
+of replicas according to the popularity.
+
+By default, this replication or eviction decision and the corresponding data transfer is completely
+transparent to users and applications accessing data stored in Alluxio.
+
+### Active Replication
+
+In addition to the dynamic replication adjustment, Alluxio also provides APIs and command-line
+interfaces for users to maintain a target range of replication level for a file explicitly.
```

Comment:
1.) replication level**s**
2.) I think it sounds better if we say "...users to **explicitly** maintain a target range of replication level**s** for a file" or we could just drop "explicitly"